query-engine/connector-test-kit-rs/query-tests-setup/src/config.rs

```diff
@@ -27,6 +43,11 @@ pub struct TestConfig {
     /// Env key: `EXTERNAL_TEST_EXECUTOR`
     external_test_executor: Option<String>,
 
+    /// Specifies which engine to use within external test executor.
+    /// If omitted, NAPI engine will be used.
+    /// Env key: `EXTERNAL_TEST_EXECUTOR_ENGINE`
+    external_test_executor_engine: Option<TestExecutorEngine>,
```

Comment:
Left a not on one of the tests config files, about using only `external_test_executor`, and making it a choice between `Napi` or `Wasm`. FWIW it's unlikely that we use a different executor than the default.